tests/configs/test_posfilter.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+import pytest
+
+from gnatss.configs.io import InputData
+from gnatss.configs.posfilter import PositionFilter, PositionFilterInputs
+
+
+def test_position_filter_input():
+    # Create test file
+    test_file = Path(__file__).parent / "test_data.csv"
```

Comment:
Can we move this to `@pytest.fixture` and also the test_file created in `test_env_main_posfilter`?